ddl.dic

```diff
@@ -1022,6 +1050,75 @@ save_enumeration_default.value
 
 save_
 
+save_ENUMERATION_DEFAULTS
+
+    _definition.id                ENUMERATION_DEFAULTS
+    _definition.scope             Category
+    _definition.class             Loop
+    _definition.update            2023-06-01
+    _description.text
+;
+    Loop of pre-determined default enumeration values indexed to a
+    data item by the (potentially) multiple keys in the item
+    _enumeration.def_index_ids.
+;
+    _name.category_id             ENUMERATION
+    _name.object_id               ENUMERATION_DEFAULTS
+    _category_key.name            '_enumeration_defaults.index'
+
+save_
+
+save_enumeration_defaults.index
+
+    _definition.id                '_enumeration_defaults.index'
+    _definition.class             Attribute
+    _definition.update            2023-06-01
+    _description.text
+;
+    Index key in the list default values referenced to by the value of
+    _enumeration.def_index_ids. The values given must be in the same order as
+    specified by _enumeration.def_index_ids.
+;
+    _name.category_id             enumeration_defaults
+    _name.object_id               index
+    _type.purpose                 Encode
+    _type.source                  Assigned
+    _type.container               Array
+    _type.dimension               '[]'
+    _type.contents                Code
+    _method.purpose               Definition
+    _method.expression
+;
+    _enumeration_defaults.index = [ _enumeration_default.index ]
```

Comment:
This dREL can't work. It states a relationship that must be true for every row, except there is no way to indicate which row of `enumeration_default` (a different category with lots of rows) you want. The `dataname[<keyvalue?]` syntax could be used, but they you don't know the value of the key yet because that's what you're calculating.

Reply:
This was resolved by removing the dREL altogether.